pysiaf/utils/compare.py

```diff
@@ -28,6 +28,22 @@
 from ..aperture import compare_apertures
 
 
+# Show plot only if in ipython session, if not in ipython always save plot
+# Either in the specified location, or in the current directory
+def show_save_plot(report_dir):
```

Comment:
I suggest to break this up into a function `is_ipython` located in the utils directory so that it can be reused.
Also I suggest to save into `os.environ['HOME']` if report_dir is None

Reply:
Done